.github/workflows/scheduled.yml

```diff
@@ -992,6 +992,7 @@ jobs:
                 --stderrthreshold=2 \
                 --log_dir=/tmp/presto_only_bias_function_fuzzer_repro/logs \
                 --repro_persist_path=/tmp/presto_only_bias_function_fuzzer_repro \
+                --velox_fuzzer_max_level_of_nesting=4 \
```

Comment:
I think it will be better if we dont change it. The failure found in array_top_n is a good thing and it will only retrigger if array_top_n is modified again. We should aim to find more such bugs.

Reply:
One thing to be mindful of is that this will fail the fuzzer and prevent potential correctness issues to be tested, so if we encounter this in the future, we should encourage the author to run this fuzzer with reduced level of nesting in order to surface actual correctness bugs